ui/component.js

```diff
@@ -2904,14 +2904,53 @@ var Component = exports.Component = Target.specialize(/** @lends Component.proto
      * Stores values that need to be set on the element. Cleared each draw cycle.
      * @private
      */
-     __elementAttributeValues: {
-         value: null
-     },
-     _elementAttributeValues: {
-         get: function() {
-             return this.__elementAttributeValues || (this.__elementAttributeValues = {});
-         }
-     },
+    __bufferedElementsAttributeValues: {
+        value: null
+    },
+
+    _bufferedElementsAttributeValues: {
+        get: function () {
+            return this.__bufferedElementsAttributeValues || (this.__bufferedElementsAttributeValues = new Map());
+        }
+    },
+
+    bufferElementAttributeValue: {
```

Comment:
let's rename this setElementAttributeValue, as by definition, Components buffer DOM Changes